storage/rocksdb/ha_rocksdb.cc

```diff
@@ -643,6 +659,60 @@ static int rocksdb_force_flush_memtable_and_lzero_now(
   return num_errors == 0 ? HA_EXIT_SUCCESS : HA_EXIT_FAILURE;
 }
 
+static void rocksdb_compact_lzero_now_stub(
+    THD *const thd MY_ATTRIBUTE((__unused__)),
+    struct SYS_VAR *const var MY_ATTRIBUTE((__unused__)),
+    void *const var_ptr MY_ATTRIBUTE((__unused__)),
+    const void *const save MY_ATTRIBUTE((__unused__))) {}
+
+static int rocksdb_compact_lzero_now(
+    THD *const thd MY_ATTRIBUTE((__unused__)),
+    struct SYS_VAR *const var MY_ATTRIBUTE((__unused__)),
+    void *const var_ptr MY_ATTRIBUTE((__unused__)),
+    struct st_mysql_value *const value) {
+  bool parsed_value = false;
+
+  if (mysql_value_to_bool(value, &parsed_value) != 0) {
+    return 1;
+  } else if (!parsed_value) {
+    // Setting to OFF is a no-op and this supports mtr tests
+    return HA_EXIT_SUCCESS;
+  }
+
+  return rocksdb_compact_lzero();
+}
+
+static void rocksdb_force_flush_memtable_and_lzero_now_stub(
+    THD *const thd MY_ATTRIBUTE((__unused__)),
```

Comment:
nit: could you omit "thd MY_ATTRIBUTE((__unused__))"? if you don't specify parameter name, c++ treat as unused parameters.

Reply:
This is the style used by existing code. I prefer to match it.